app/helpers/events_helper.rb

```diff
@@ -10,5 +10,13 @@ def event_title_line(event)
     t(:event_title_line, :name => event.name, :start => l(event.start_date),
       :end => l(event.end_date), :country => event.country_code)
   end
-end
 
+  def users_for_event(state)
+  	if state != 'all'
```

Comment:
Please use 2 whitespaces for indentation, and no tabs.

We don't follow a particular style guide right now, but [this one](https://github.com/bbatsov/ruby-style-guide) is widely used in ruby world and might be a good reference.